docs/src/add-services/redis.md

```diff
@@ -189,62 +184,84 @@ const value = await client.get('x'); // returns 42
 
 {{< /codetabs >}}
 
+## Relationship
+
+After you've [configured your Redis service](#usage-example),
+you can retrieve information such as the credentials and port you need to connect to your Redis service.
+
+To do so, you can access the [`PLATFORM_RELATIONSHIPS` environment variable](../development/variables/use-variables.md#use-platformsh-provided-variables) directly [in your app](../../development/variables/use-variables.md#access-variables-in-your-app),
+for example using a Platform.sh configuration reader.
```

Comment:
:hammer_and_wrench: This section should be just a reference and this intro seems rather long for that. We have [suggested wording for an intro](https://github.com/platformsh/platformsh-docs/blob/main/docs/templates/service-reference.md?plain=1#L75-L76). Do you think that's missing something?